cordova/cordova-sampleapp-android-tests/sampleapp/comm.py

```diff
@@ -181,7 +181,7 @@ def create(appname, pkgname, mode, sourcecodepath, replace_index_list, self, ext
         do_remove(glob.glob(os.path.join(project_root, "www")))
         do_copy(sourcecodepath, os.path.join(tool_path, appname, "www"))
 
-def buildGoogleApp(appname, sourcecodepath, self):
+def buildGoogleApp(appname, sourcecodepath, self, cordova_android_ver = "6.0.0"):
```

Comment:
I don't think make default value "6.0.0" is a good point, just ignore it. And pass specific version in the param.